src/cfnlint/rules/__init__.py

```diff
@@ -136,6 +136,8 @@ def wrapper(self, filename, cfn, *args, **kwargs):
 
                 if results:
                     for result in results:
+                        if 'Fn::Transform' in result.path:
```

Comment:
Assuming we've expanded [short form to long form](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-transform.html#intrinsic-function-reference-transform-declaration) by this point?

Reply:
Good question. Hmm.  Shouldn't the short form be `!Transform`?